.travis.yml

```diff
@@ -1,15 +1,21 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
-  - 7.0
+  - '5.5'
+  - '5.6'
+  - '7.0'
   - hhvm
 
 matrix:
   allow_failures:
-    - php: 7.0
+    - php: '7.0'
```

Comment:
I've not seen it before, all the projects I've seen leave it without the quotes...